src/styles/blocks/social-list.css

```diff
@@ -0,0 +1,38 @@
+.social-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.social-list__link {
+    display: flex;
```

Comment:
Please, make it `inline-flex`. It's better when the link has no extra width on mouse events.